### PR TITLE
New MET filters (part 1 of #16174 with #17720 developments) for 80X

### PR DESCRIPTION
**Pull request by mmarionncern**
Following the discussions of #16174 and the development made for 90X in #17720 , this packages contains only the new MET filters needed for the reprocessing v1.

- new filters added
- configuration modified

matrix tests successful

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by mmarionncern** [comment]
Ping?

**Comment by slava77** [comment]
integration/review of this PR was awaiting the 9X PR integration

**Comment by mmarionncern** [comment]
Thanks for the explanation, sorry for the noise

File: RecoMET/METFilters/plugins/BadGlobalMuonTagger.cc [hunk 1]
```diff
@@ -1,130 +0,0 @@
-#include "FWCore/Framework/interface/stream/EDFilter.h"
```
Comment (slava77) [inline RecoMET/METFilters/plugins/BadGlobalMuonTagger.cc]: why is this file removed?
It's still used in the bad muon mitigation machinery.
was it just dropped by mistake?

Comment (mmarionncern) [inline RecoMET/METFilters/plugins/BadGlobalMuonTagger.cc]: looks like a mistake

File: RecoMET/METFilters/python/metFilters_cff.py [hunk 2]
```diff
@@ -73,6 +73,19 @@
...
+
+## The charged hadron track track filter (2016) ____________________________________||
+from RecoMET.METFilters.BadChargedCandidateFilter_cfi import *
+
+## The muon bad track filter (2016) ________________________________________________||
+from RecoMET.METFilters.BadPFMuonFilter_cfi import *
+
+
 metFilters = cms.Sequence(
    HBHENoiseFilterResultProducer *
```
Comment (slava77) [inline RecoMET/METFilters/python/metFilters_cff.py]: the new Bad\* filters are not included in this sequence (while they are in 91X version).
Is this intentional here?

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/17968	ecc32e6
- new MET filters  are implemented consistent with 91X version available in #18165; the changes are also in the scope of miniAODv1 as being plain additions and relevant mostly for data
- jenkins tests pass and comparisons with baseline show no differences
- local tests confirm that the 4 new filters run and consume under 1 ms/event total
